CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - [#871](https://github.com/plotly/dash-core-components/pull/871) Add Julia syntax highlighting support for dcc.Markdown
 
 ## Updated
+- [#878](https://github.com/plotly/dash-core-components/pull/878)
+  - used the `getTooltipContainter` prop of dcc.Slider to make sure tooltips are positioned correctly when the parent slider is not visible due to it being in a HTML element with an `overflow-x` style, fixing [#751](https://github.com/plotly/dash-core-components/issues/751)
```

Comment:
I think I missed the wrong level of details of https://github.com/plotly/dash-core-components/blame/dev/CHANGELOG.md#L26 which you are using as your baseline.

Make the entry for focused on what this fixes for the user. Something along the lines of:

Fix a bug causing the dcc.Slider and dcc.RangerSlider tooltip to be visible even if the component isn't visible (e.g. overflow)